MAINTAINERS.md

```diff
@@ -5,7 +5,13 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 ## Current Maintainers
 
 | Maintainer              | GitHub ID                                   | Affiliation |
-| ----------------------- | ------------------------------------------- | ----------- |
-| Shivam Dhar             | [Shivamdhar](https://github.com/Shivamdhar) | Amazon      |
+|-------------------------|---------------------------------------------| ----------- |
 | Vijayan Balasubramanian | [VijayanB](https://github.com/VijayanB)     | Amazon      |
 | Vamshi Vijay Nakkirtha  | [vamshin](https://github.com/vamshin)       | Amazon      |
+| Junqiu Lei              | [junqiu-lei](https://github.com/junqiu-lei) | Amazon      |
+
+## Emeritus
+
+| Maintainer              | GitHub ID                                   | Affiliation |
+|-------------------------|---------------------------------------------| ----------- |
+| Shivam Dhar             | [Shivamdhar](https://github.com/Shivamdhar) | Amazon      |
```

Comment:
just double-checking, we want to keep Shivam as maintainer, right?

Reply:
No, as Emeritus.

Reply:
No, but we can move Shivam here. Ref: https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#moving-on

Reply:
Got it, didn't know about emeritus section. And how about Affiliation, should we keep Amazon?